python/itools-y4m.py

```diff
@@ -62,37 +55,34 @@ def do_range_conversion(inarr, srcmin, srcmax, dstmin, dstmax):
     # look for invalid values
     broken_range = False
     if (
-        len(outarr[outarr < np.iinfo(np.uint8).min]) > 0
-        or len(outarr[outarr > np.iinfo(np.uint8).max]) > 0
+        len(outarr[outarr < dstmin]) > 0
```

Comment:
Here you're fixing a bug. You're checking for the output not being in the expected dst range, instead of the actual capacity of uint8. Please split to a separate diff

Reply:
moved into a different commit